final_prep/annotations.rb

```diff
@@ -19,25 +20,27 @@ def build_a_bear(name, age, fur, clothes, special_power)
   return built_bear
 end
 
+# Makes two instance variables with the build_a_bear object
```

Comment:
These are not instance variables. A better way to annotate this would be to say "Invoke a function called build_a_bear with 4 arguments"